server/controllers/recipeController.js

```diff
@@ -103,12 +103,17 @@ const updateRecipe = asyncHandler( async (req, res) => {
 })
 
 const viewShoppingList = asyncHandler( async (req, res) => {
-	const recipeList = await getShoppingList(req.user.id, Recipe, Ingredient);
-	const shoppingList = recipeList
+	const shoppingList = await getShoppingList(req.user.id, Recipe, Ingredient);
```

Comment:
not sure about this but I believe `req.user.id` will return null, it should be `req.user.dataValues.id`. Also for `viewShoppingListRecipes`